app/Console/Commands/Command.php

```diff
@@ -206,4 +216,35 @@ protected function getAuthorLink()
             return 'https://gitlab.com/vatsim-uk/core/blob/development'.$directory;
         }
     }
+
+    private function handleSlackException($e)
+    {
+        $errorClass = get_class($e);
+        $errorCode = $e->getCode();
+        if ($errorClass == ClientException::class) {
```

Comment:
I may be being a little thick here but what is the purpose of checking the error class? Can we not just grab the code, if 408 or 504, break else BugSnag?

Reply:
Good point. I will make it catch the `GuzzleHttp\Exception\BadResponseException` which will catch both.